builder/requirements.txt

```diff
@@ -0,0 +1,14 @@
+pyarrow
+pandas
+anndata
+numpy
+tiledb
+tiledbsoma
```

Comment:
```suggestion
# NOTE: Until this code and TileDB-SOMA are stable, you may need to build TileDB-SOMA locally, per ./notebooks/README.md.
tiledbsoma
```

Reply:
I will add this, and comment out the tiledbsoma - until it _is_ available on PyPi (in a week-ish)